lib/widgets/rules_buttons.dart

```diff
@@ -0,0 +1,102 @@
+import 'package:flutter/material.dart';
+
+import 'game_view.dart';
+
+class RulesChangingButtons extends StatefulWidget {
+  const RulesChangingButtons({super.key});
+
+  @override
+  RulesChangingButtonsState createState() => RulesChangingButtonsState();
+}
+
+class RulesChangingButtonsState extends State<RulesChangingButtons> {
+  @override
+  Widget build(BuildContext context) {
+    return Row(
+      mainAxisAlignment: MainAxisAlignment.center,
+      children: [
+        const Text('Get'),
+        const SizedBox(width: 20),
+        SizedBox(
```

Comment:
This is a big code duplicate.
Can you create a widget like RoundedDropdown, that will take items and onChanged as parameters, and use it two times?

Reply:
Done